app/boards/shields/splitkb_aurora_lily58/splitkb_aurora_lily58.dtsi

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+
+	chosen {
+		zephyr,display = &oled;
+		zmk,matrix_transform = &default_transform;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <16>;
```

Comment:
```suggestion
		columns = <14>;
```
I think this doesn't do anything at the moment but should this be 14?

Reply:
Indeed. Fixed.